tests/integration/route_test.py

```diff
@@ -1020,6 +1020,31 @@ def test_route_rule_from_all_to_all_ipv6(route_rule_test_env):
         assert _check_ip_rules(rules)
 
 
+@pytest.mark.tier1
+@pytest.mark.skipif(
+    nm_minor_version() < 42, reason="Loopback is only support on NM 1.42+"
+)
+def test_route_rule_add_and_remove_using_loopback():
+    desired_state = {
```

Comment:
I noticed that you did not specify any device in the `desired_state`, does that mean `RouteRule.IIF` is the loopback device by default ?

Reply:
It is allowed to have empty `RouteRule.iif`. The loopback interface is the final fallback on choosing interface to store route rule. You make check `store_route_rule_config()` for detail.